backend/schema/field.go

```diff
@@ -12,24 +12,24 @@ import (
 type Field struct {
 	Pos Position `parser:"" protobuf:"1,optional"`
 
-	Comments []string `parser:"@Comment*" protobuf:"3"`
-	Name     string   `parser:"@Ident" protobuf:"2"`
-	Type     Type     `parser:"@@" protobuf:"4"`
-	Alias    string   `parser:"('alias' @Ident)?" protobuf:"5"`
+	Comments  []string `parser:"@Comment*" protobuf:"3"`
+	Name      string   `parser:"@Ident" protobuf:"2"`
+	Type      Type     `parser:"@@" protobuf:"4"`
+	HTTPAlias string   `parser:"('alias' 'http' @Ident)?" protobuf:"5"`
```

Comment:
Just want to check that we're ok with this new `HTTPAlias` name.

Reply:
Also, @alecthomas does this parser change look correct to you? I sort of just guessed 😬

Reply:
Looks good!